SECURITY.md

```diff
@@ -0,0 +1,14 @@
+# Security Policy
+
+## Supported Versions
+
+We generally only fully support the latest version, the same applies to exploits such as server crashes and item
+duplication bugs. In the transition period during larger Minecraft updates, we may still backport important fixes to the
+last minor or major release.
+
+## Reporting a Vulnerability
+
+For any issues that are NOT duplication bugs, server/client crashes, or otherwise serious exploits, please open an issue
+through the [Issues tab](https://github.com/PaperMC/Paper/issues).
+For exploits, please [join our Discord](https://discord.gg/papermc) and see the `#paper-exploit-report` channel for
```

Comment:
I think we can link to the channel directly too. Recently I've noticed that clicking on discord links that open browsers offer an option to open it in the app (finally) so linking to discord from outside discord is fine.

Reply:
can be good but need remember if user not confuse for the rare case where they are not in the server and discord just show missing access.